### DISCUSSION TITLE
Release 0.3.2 (1.19.80.22)

### DISCUSSION THREAD

**Discussion by minecraftchloe** [Mojang/minecraft-editor#95] [Announcements]
_[Read the previous release notes here](https://github.com/Mojang/minecraft-editor/discussions/86)_

---

## Summary
Thanks again for another incredible week. We are blown away by all the wonderful feedback that we have received so far from you!  Please keeping sending any and all feedback you have.  We very much appreciate it!

_Disclaimer:_ As our small team is heads-down working towards v0.4: The API Update, which will include documentation and a starter-kit, any bug and quality fixes week to week will be very modest in size and scope. 

---

## Fixed this week

- Fixed issues with UI not scaling properly when the resolution or game window size is updated.   Previously the UI would remain the same size when adjusting the game window size or changing the screen resolution. Let us know what you think! Help us find some edge cases.
- Item drops have been disabled in Editor. This allows for Undo/Redo and Cut/Copy/Paste to work properly. Items will no longer be left floating in the world.  Items can’t be dropped in Tool mode or Crosshair mode.  Any block that doesn’t adhere to the Minecraft placement rules will instantly disappear.  This goes for copy/paste, brushes, or pressing Q to drop an item from your hotbar.
- Fixed a bug [#56 / #92 ] where Tool mode would use the 3rd person camera if it was the active camera. when switching from Crosshair mode. The 1st person camera is always used in Tool mode.  Crosshair mode will still remember your state and return to your previous setting when switch back to Crosshair mode.
- Fixed issues [#66 ] where camera control was disabled when using a Nether or End Portal in Crosshair mode.  Mouse retains camera control when going through portals.
- Removed the old stonecutter and Nether reactor core from the block picker.  These blocks were left in by mistake and are only available to ensure older content still functions.  All new content should be made without using the deprecated blocks, please 😅!

---

## Next
The Preview branch is going to be locked for critical bug fixes the next few weeks. Once the Preview branch opens up again, we will post Editor updates!

v0.4: The API Update will be focused on exposing the Editor APIs for you to begin building your own Editor extensions. For now we invite you to acquaint yourself with the existing [scripting](https://docs.microsoft.com/en-us/minecraft/creator/scriptapi/) efforts and get familiar with Typescript!

Additionally, if you're new to the Editor GitHub this week, we encourage you to join the Bedrock Add-Ons Discord [#Editor channel](https://discord.gg/GepAEExA)!

---

Thank you for being here! You're the best! ❤️

**Comment by MetalManeMc** [comment]
It would be nice if these old blocks could still be used if you input their ID, even though removed from the dropdown block list.
It would be a nice easter egg 😉

**Reply by ScoutTheSaneless** [reply]
Or even a toggle somewhere to allow them into the drop-downs, which could also include a toggle to include EDU features
